### PR TITLE
Add new is available field to blobber

### PR DESCRIPTION
**Pull request by Sriep**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk: https://github.com/0chain/gosdk/pull/866
- system_test: https://github.com/0chain/system_test/pull/612
- zboxcli: https://github.com/0chain/zboxcli/pull/398
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | blobber_available  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

File: code/go/0chain.net/smartcontract/storagesc/blobber.go [hunk 1]
```diff
@@ -170,8 +170,8 @@ func (sc *StorageSmartContract) updateBlobber(t *transaction.Transaction,
 	if err = sp.Save(spenum.Blobber, blobber.ID, balances); err != nil {
 		return fmt.Errorf("saving stake pool: %v", err)
 	}
-
-	if err := emitAddOrOverwriteBlobber(blobber, sp, balances); err != nil {
+	logging.Logger.Info("piers updateBlobber end", zap.Any("blober", blobber))
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/blobber.go]: Remove debug logging before merging

File: code/go/0chain.net/smartcontract/storagesc/blobber.go [hunk 2]
```diff
@@ -291,6 +292,7 @@ func (sc *StorageSmartContract) updateBlobberSettings(t *transaction.Transaction
 		return "", common.NewError("update_blobber_settings_failed",
 			"malformed request: "+err.Error())
 	}
+	logging.Logger.Info("piers updateBlobberSettings", zap.Any("update blobber", updatedBlobber))
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/blobber.go]: Same as above

File: code/go/0chain.net/smartcontract/storagesc/blobber_eventdb.go [hunk 3]
```diff
@@ -39,7 +40,7 @@ func emitAddOrOverwriteBlobber(sn *StorageNode, sp *stakePool, balances cstate.S
 		},
 		OffersTotal: sp.TotalOffers,
 	}
-
+	logging.Logger.Info("piers emitUpdateBlobber", zap.Any("data", data))
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/blobber_eventdb.go]: Same

**Review by peterlimg — CHANGES_REQUESTED** [review]
Just a few minor change requests

**Review by peterlimg — APPROVED** [review]
LGTM

**Pull request merged**